File: TopInterview150/RemoveDuplicatesFromSortedArray2/solution.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        if (nums.size() > 2)
        {
            int counter = 2;
            for (int i = 2; i<nums.size(); i++)
            {
                if (nums[i] != nums[counter-1] || nums[i] != nums[counter-2])
                {
                    nums[counter] = nums[i];
                    counter++;
                }
            }
            return counter;
        }
        else
        {
            return nums.size();
        }
    }
};

File: TopInterview150/RemoveDuplicatesFromSortedArray2/solution.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        counter = 2
        """for x in range(start, end, stepValue)"""
        for x in range(2, len(nums), 1):
            if (nums[x] != nums[counter-1] or nums[x] != nums[counter-2]):
                nums[counter] = nums[x]
                counter+=1
        return counter
